test/run-openshift

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+#
+# Test the NodeJS image in OpenShift.
+#
+# IMAGE_NAME specifies a name of the candidate image used for testing.
+# The image has to be available before this script is executed.
+#
+
+THISDIR=$(dirname ${BASH_SOURCE[0]})
+
+source ${THISDIR}/test-lib.sh
+source ${THISDIR}/test-lib-openshift.sh
+
+set -exo nounset
+
+test -n "${IMAGE_NAME-}" || false 'make sure $IMAGE_NAME is defined'
+test -n "${VERSION-}" || false 'make sure $VERSION is defined'
+
+ct_os_cluster_up
+# TODO: Can we make the build against examples inside the same PR?
+ct_os_test_s2i_app ${IMAGE_NAME} "https://github.com/sclorg/s2i-nodejs-container.git" test/test-app "This is a node.js echo service"
```

Comment:
Same issue as with perl, a lot of log pollution when getting the response. Might be better to fix this inside common-container-scripts though (saving the response into a temporary file instead of a variable?)

Reply:
Thanks for the suggestion, should be fixed now by https://github.com/sclorg/container-common-scripts/commit/f51071814e57dc0fd118c8dc961fd6971b09db76

Reply:
Looks much better now, thanks!